L1Trigger/L1TGlobal/plugins/L1TGlobalProducer.cc

```diff
@@ -556,6 +570,9 @@ void L1TGlobalProducer::produce(edm::Event& iEvent, const edm::EventSetup& evSet
   // * produce the GlobalObjectMapRecord
   std::unique_ptr<GlobalObjectMapRecord> gtObjectMapRecord(new GlobalObjectMapRecord());
 
+  // if (m_saveAxoScore)
+  std::unique_ptr<AXOL1TLScoreBxCollection> uGtAXOScoreRecord(new AXOL1TLScoreBxCollection(maxEmulBxInEvent));
```

Comment:
As I understand it, the method is defined here https://github.com/cms-sw/cmssw/blob/6a131029e88625000a80f6618e4a173d325fffc0/DataFormats/L1TGlobal/interface/AXOL1TLScore.h and I want to give it the argument of number of bx in events so that I can set the score to zero if it is not at bx=0, modeled after the globalalgblk method https://github.com/cms-sw/cmssw/blob/6a131029e88625000a80f6618e4a173d325fffc0/DataFormats/L1TGlobal/interface/GlobalAlgBlk.h I will take another look and make sure it makes sense